subsys/bluetooth/audio/Kconfig.pacs

```diff
@@ -44,6 +50,13 @@ config BT_PAC_SNK_LOC_WRITEABLE
 	  This option enables support for clients to write to the Sink PAC
 	  Location Characteristic.
 
+config BT_PAC_SNK_LOC_NOTIFIABLE
+	bool "Sink Audio Location Notifiable Support"
+	depends on BT_PAC_SNK_LOC
+	help
+	  This option enables support for clients to be notified on the Sink
+	  Audio Location Characteristic.
```

Comment:
```suggestion
	  This option enables support for clients to be notified on the Sink
	  Audio Location changes.
```

Reply:
For all of these I prefer then to change it to 'Audio Location Characteristic changes.', since we everywhere else have it mentioned explictely as a characteristic